cmd/bbolt/command_get.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+func newGetCommand(m *Main) *cobra.Command {
+	var parseFormat, format string
+
+	cmd := &cobra.Command{
+		Use:   "get PATH [BUCKET..] KEY",
+		Short: "Get the value of a key from a (sub)bucket in a bbolt database",
+		Args:  cobra.MinimumNArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := args[0]
+			buckets := args[1 : len(args)-1]
+			keyStr := args[len(args)-1]
+
+			key, err := parseBytes(keyStr, parseFormat)
+			if err != nil {
+				return err
+			}
+			if path == "" {
+				return ErrPathRequired
+			} else if _, err := os.Stat(path); os.IsNotExist(err) {
+				return ErrFileNotFound
+			} else if len(buckets) == 0 {
```

Comment:
so i am not imposing any approach, but one benefit of this example https://github.com/etcd-io/bbolt/blob/main/cmd/bbolt/command_buckets.go, is that you can make these checks wthin `RunE()`, away from the actual logic

e.g. 

```
RunE: func(cmd *cobra.Command, args []string) error {

                              if len(key) == 0 {
				return fmt.Errorf("key required")
			}

			return bucketsFunc(cmd, args[0])
		},
```